core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/coprocessor/transactional/TrxRegionEndpoint.java

```diff
@@ -2911,14 +2918,28 @@ public void start(CoprocessorEnvironment env) throws IOException {
         }
 
         this.scannerLeaseTimeoutPeriod = HBaseConfiguration.getInt(config,
-          HConstants.HBASE_CLIENT_SCANNER_TIMEOUT_PERIOD,
-          HConstants.HBASE_REGIONSERVER_LEASE_PERIOD_KEY,
-          HConstants.DEFAULT_HBASE_CLIENT_SCANNER_TIMEOUT_PERIOD);
+								   HConstants.HBASE_CLIENT_SCANNER_TIMEOUT_PERIOD,
+								   HConstants.HBASE_REGIONSERVER_LEASE_PERIOD_KEY,
+								   HConstants.DEFAULT_HBASE_CLIENT_SCANNER_TIMEOUT_PERIOD);
+
         this.scannerThreadWakeFrequency = config.getInt(HConstants.THREAD_WAKE_FREQUENCY, 10 * 1000);
 
         this.cleanTimer = config.getInt(SLEEP_CONF, DEFAULT_SLEEP);
         this.memoryUsageThreshold = config.getInt(MEMORY_THRESHOLD, DEFAULT_MEMORY_THRESHOLD);
         this.memoryUsagePerformGC = config.getBoolean(MEMORY_PERFORM_GC, DEFAULT_MEMORY_PERFORM_GC);
+        this.asyncWal = config.getInt(CONF_ASYNC_WAL, DEFAULT_ASYNC_WAL);
+        this.skipWal = config.getBoolean(CONF_SKIP_WAL, DEFAULT_SKIP_WAL);
+        this.fullEditInCommit = config.getBoolean(CONF_COMMIT_EDIT, DEFAULT_COMMIT_EDIT);
+	m_regionName = this.regionInfo.getRegionNameAsString();
+	m_isTrafodionMetadata = m_regionName.contains("_MD_");
```

Comment:
This could be fooled, right? If my table name was _MD_, this would return true? Also if my schema name was ABC_MD_DEF?